WPFSKillTree/Locale/ru/Messages.po

```diff
@@ -80,8 +81,7 @@ msgstr "OK"
 #: Controls\Dialogs\Views\MetroMessageBoxView.xaml:76
 #: Controls\Dialogs\Views\ValidatingInputDialogView.xaml:49
 #: Views\Builds\EditBuildWindow.xaml:48 Views\Crafting\CraftingView.xaml:155
-#: Views\Crafting\UniqueCraftingView.xaml:62
-#: Views\Equipment\TabPicker.xaml:127
+#: Views\Crafting\UniqueCraftingView.xaml:62 Views\Equipment\TabPicker.xaml:127
```

Comment:
Looks like you accidentally deleted a new line here

Reply:
Fixed